rfcs/012 - Either, Option, and Result Destructuring.md

```diff
@@ -12,7 +12,7 @@ YYYY-MM-DD
 
 #### Approvers
 
-- Full Name <email@example.com>
+- David Ellis david@alantechnologies.com
```

Comment:
Please put the `<>` around the email. :) It makes the email address actually clickable with a `mailto:email@address` syntax in your browser.